### PR TITLE
docs: add how to integrate with grafana preview link

### LINKED ISSUE #1699: after configuring the Grafana address in Metrics, the page cannot be displayed normally
# Feature request

## Please describe your feature

After many users configure the Grafana address in `Metrics`, the following situation is prone to appear:
image

Although this is caused by the cross-domain configuration of the Grafana service and the anonymous login has not been turned off. However, I think we should provide a solution, such as documenting a solution to this situation in the FAQ document.

## Describe the solution you'd like

I think we should provide a solution, such as documenting a solution to this situation in the FAQ document.

What do you think.

### PR DESCRIPTION
**Pull request by wgy-home**
Precautions for configuring grafana.

Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

fix #1699 

Please update this section with detailed description.

**Related issues**

fix/resolve #0001

**Checklist:**

- [ ] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by wgy-home** [comment]
Add FAQ.

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -70,3 +70,18 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
 ```shell
 $ swagger-markdown -i ./docs/en/latest/api/api.yaml
 ```
+### 6. Apisix dashbord add grafana cross domain problem
```
Comment (juzhiyuan) [inline docs/en/latest/FAQ.md:73]:
```suggestion
### 6. APISIX dashboard add grafana cross-domain problem
```

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -70,3 +70,18 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
...
+Modifying the grafan configuration
```
Comment (juzhiyuan) [inline docs/en/latest/FAQ.md:74]:
```suggestion
Modifying the Grafana configuration:

```

File: docs/en/latest/FAQ.md [hunk 3]
```diff
@@ -70,3 +70,18 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
...
+1. Allow anonymous access. enabled = true
```
Comment (juzhiyuan) [inline docs/en/latest/FAQ.md:75]:
```suggestion
1. Enable anonymous access:
```

File: docs/en/latest/FAQ.md [hunk 4]
```diff
@@ -70,3 +70,18 @@ $ swagger generate spec -o ./docs/en/latest/api/api.yaml --scan-models
...
+```shell
+# grep 'auth.anonymous' -A 3 defaults.ini
+[auth.anonymous]
+# enable anonymous access
+enabled = true
+```
+2. Allow access via iframe. allow_embedding = true
```
Comment (juzhiyuan) [inline docs/en/latest/FAQ.md:82]:
```suggestion
2. Allow access via iframe
```

**Review by juzhiyuan — COMMENTED** [review]
Wow! Thanks a lot, and there have some minor changes that would make this PR better.

**Comment by juzhiyuan** [comment]
Hi @wgy-home, I noticed there have some lint errors, would you like to fix them?

image

**Review by liuxiran — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Comment by iamayushdas** [comment]
Hey @wgy-home could you please follow semantic format for PR title. 
docs: "your pr title"
Thanks for contributing

**Review by iamayushdas — APPROVED** [review]

File: docs/en/latest/FAQ.md [hunk 5]
```diff
@@ -109,3 +109,23 @@ Note: You can use this method in development and test environment to allow all I
...
+# enable anonymous access
+enabled = true
+```
+
+2. Allow access via iframe
+
+```shell
+# grep 'allow_embedding' defaults.ini
+allow_embedding = true
+···
```
Comment (starsz) [inline docs/en/latest/FAQ.md:131]: Maybe we should enable `cors` in grafana config?

**Comment by starsz** [comment]
By the way, I think you should change the title of this PR.

**Pull request merged**